src/transformStateWithClones.js

```diff
@@ -7,7 +7,37 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  const stateVersion = { ...state };
+
+  for (const action in actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
Consider using an appropriate cycle for arrays